<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrateDialog</class>
 <widget class="QTabWidget" name="CalibrateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>校准数据</string>
   </attribute>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>326</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="lstView_ItemName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <property name="rowWrapPolicy">
        <enum>QFormLayout::DontWrapRows</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 85, 255);</string>
         </property>
         <property name="text">
          <string>校准1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edt_Calibrate0"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 85, 255);</string>
         </property>
         <property name="text">
          <string>校准2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edt_Calibrate1"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 85, 255);</string>
         </property>
         <property name="text">
          <string>校准3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edt_Calibrate2"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 85, 255);</string>
         </property>
         <property name="text">
          <string>校准4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="edt_Calibrate3"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 85, 255);</string>
         </property>
         <property name="text">
          <string>校准5</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="edt_Calibrate4"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 85, 255);</string>
         </property>
         <property name="text">
          <string>校准6</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="edt_Calibrate5"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 85, 255);</string>
         </property>
         <property name="text">
          <string>校准7</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="edt_Calibrate6"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 85, 255);</string>
         </property>
         <property name="text">
          <string>校准8</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="edt_Calibrate7"/>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="btn_CalibrateSave">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/image/Save.png</normaloff>:/image/Save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="edt_LaderValue7">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>234</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_LaderValue2">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>68</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_LaderValue4">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>134</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_LaderValue5">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>168</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_LaderValue3">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>100</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>234</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>梯度值8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>33</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>梯度值2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>梯度值1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>168</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>梯度值6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>201</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>梯度值7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>134</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>梯度值5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>68</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>梯度值3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_LaderValue0">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>0</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>100</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>梯度值4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_LaderValue6">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>201</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_LaderValue1">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>33</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>校准观察</string>
   </attribute>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="rowWrapPolicy">
          <enum>QFormLayout::WrapAllRows</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="horizontalSpacing">
          <number>3</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="label_20">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 2, 2);
font: 11pt &quot;Vemana2000&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;校准项目&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cbb_ItemName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <item>
            <property name="text">
             <string>BIL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>URO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KET</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BLD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PRO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NIT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WBC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GLU</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PH</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>VC</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 2, 2);
font: 11pt &quot;Vemana2000&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;计算校准值&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lstResultView"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tb_CalibrateOberserve">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">alternate-background-color: rgb(255, 170, 0);
color: rgb(0, 0, 255);
font: 63 11pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_NextPage">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_PreviousPage">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>前一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_FirstPag">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>首页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_LastPage">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>最后一页</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_CalibrateData">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>WenQuanYi Zen Hei</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>定标</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/image/Query.png</normaloff>:/image/Query.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_CalibrateStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>WenQuanYi Zen Hei</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
       <property name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/image/Query.png</normaloff>:/image/Query.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbTotalCount">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>314</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTotalCount_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>312</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>共</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTotalCount_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>312</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>页</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
